examples/player/simid_player.js

```diff
@@ -431,7 +431,10 @@ class SimidPlayer {
     this.videoTrackingEvents_.set("durationchange", () => {
       this.simidProtocol.sendMessage(MediaMessage.DURATION_CHANGED);
     });
-    this.videoTrackingEvents_.set("ended", this.videoComplete.bind(this));
+    // this.videoTrackingEvents_.set("ended", this.videoComplete.bind(this));
```

Comment:
remove the commented code since you have it defined better below.

Reply:
changed back to original